README.org

```diff
@@ -57,6 +57,34 @@ whether a directory is the project root:
   (setq pytest-project-root-test (lambda (dirname) (equal dirname "foo")))
 #+END_SRC
 
+By default, test paths passed to pytest are absolute paths:
+#+BEGIN_SRC sh
+  cd /project/root && pytest /base/path/relative/path/to/test.py
+#+END_SRC
+
+To override this behavior, set an optional source root directory:
```

Comment:
I'd go ahead and describe your use case here b/c I think it gives a good example of how docker might be helpful and clarifies why you'd want to use relative paths.

Reply:
Makes sense, I'll add.